content/en/blog/_posts/2022-08-26-Announcing-Auto-Refreshing-Official-CVE-Feed/index.md

```diff
@@ -0,0 +1,159 @@
+fields that
+are [JSON Feed v1.1](https://www.jsonfeed.org/version/1.1/) compliant.
+
+This JSON field is then parsed into a Markdown table for easy viewing as well as
+published as RAW JSON file for programmatic consumption through command line
+tools like `curl` or `wget`.
+
+## Design Considerations
+
+Building trust and ensuring that the feed is not stale were the main aspects
+that were considered critical when designing this feature for success and
+widespread adoption of this feed.
+
+### Integrity and Access Control Protections
+
+Changes to any of the four artifacts used to build this feed, could lead to feed
+tampering, broken JSON, inconsistent or stale data.
+
+Let's look at how access is controlled for them one by one:
+
+#### GitHub Issues for Publicly announced CVEs
+
+Adding the `official-cve-feed` label is restricted to limited number of trusted
+community members. Who has access to add this label is defined
+in [this configuration file](https://github.com/kubernetes/test-infra/blob/master/config/prow/plugins.yaml#L149-L159)
+. Any updates to this configuration file require the changes to go through the
+existing [code review and approval process](https://www.kubernetes.dev/docs/guide/pull-requests/)
+
+#### Prow Configuration
+
+Prow job is defined in
+a `kubernetes/infra` [configuration file](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-k8s-infra/trusted/sig-security-trusted.yaml#L94-L115)
+The shell script to push and pull the data in Google Cloud Bucket is defined in
+`kubernetes/sig-security` [file](https://github.com/kubernetes/sig-security/tree/main/sig-security-tooling/cve-feed/hack)
+under `sig-security-tooling` sub-project. Both of these files goes through the same code review and
+approval process mentioned earlier.
+
+#### Google Cloud Bucket
+
+Write access to Google Cloud bucket is configured to be restricted to a set of
+trusted community members managed via an
+invite-only [Google Groups Membership](https://github.com/kubernetes/k8s.io/blob/main/groups/sig-security/groups.yaml)
+under the `kubernetes.io` domain.
+
+#### Website Data templates
+
+Website data templates that fetch and parse the stored JSON blob, are managed under `kubernetes/website` and have to follow
+the same code review and approval process as mentioned earlier.
+
+### Freshness Guarantees
+
+The job runs every `two` hours and compares the `sha256` checksum of the
+existing contents of the bucket with checksum of the latest JSON blob generated through GitHub issues.
+If the there is new data available the hashes do not match (typically because of
+a newly announced CVE) and the updated JSON blob is pushed onto to the bucket
+replacing the old blob and old hash checksum.
+
+If the hashes match, the `write to bucket` operation is skipped to reduce
+redundant updates to the cloud bucket. This also sets us up for more frequent runs
+of the prow job if needed in the future.
```

Comment:
How would you explain this to someone who isn't particularly technical and just wants to know if the feed might be stale?

Reply:
Tried resolving this in the other PR https://github.com/kubernetes/contributor-site/pull/330